assets/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "amethyst-assets"
+version = "0.1.0"
+authors = ["torkleyy <torkleyy@gmail.com>"]
+readme = "README.md"
+description = """
+Asynchronous asset management for games.
+"""
+
+[dependencies]
+fnv = "1"
+futures = "0.1.14"
+parking_lot = "0.4.4"
```

Comment:
Id personally prefer it if we did since there is no guarantee a dependency couldnt accidentally break something. Moreso it is just an honor code right now.